pkg/cmd/codespace/ports.go

```diff
@@ -74,13 +79,19 @@ func (a *App) ListPorts(ctx context.Context, codespaceName string, exporter cmdu
 		a.errLogger.Printf("Failed to get port names: %v", devContainerResult.err.Error())
 	}
 
-	portInfos := make([]*portInfo, len(ports))
-	for i, p := range ports {
-		portInfos[i] = &portInfo{
+	var portInfos []*portInfo
+
+	for _, p := range ports {
+		p := p
```

Comment:
This assignment wasn't necessary before; how come it's added now?

Reply:
👍🏻 you're right. its unnecessary because we arent in a closure. removed